keras/backend/cntk_backend.py

```diff
@@ -936,12 +936,23 @@ def pow(x, a):
 
 
 def clip(x, min_value, max_value):
-    if max_value is not None and max_value < min_value:
-        max_value = min_value
-    if max_value is None:
-        max_value = np.inf
+    """Element-wise value clipping.
+
+    # Arguments
+        x: Tensor or variable.
+        min_value: Python float, integer or tensor.
+        max_value: Python float, integer or tensor.
+
+    # Returns
+        A tensor.
+    """
+    if isinstance(min_value, (int, float)) and isinstance(max_value, (int, float)):
```

Comment:
Please break line into 2, for length

```python
if (isinstance(min_value, (int, float)) and
    isinstance(max_value, (int, float))):
```

(same below)

Reply:
@jacquerie I believe this is the right way to do it.